### PR TITLE
Implements Partial depth websocket that can be used as simple mean to obtain Ask and Bid for a symbol

### PR DESCRIPTION
**Pull request by N4rk0**
## Overview
Implements Partial depth websocket that can be used as simple mean to obtain Ask and Bid for a symbol.

## Solved Issues
N/A

## Installation
N/A

## Acceptance and Testing Steps
- Compatible for all targets.

### CODE REVIEW

File: BinanceExchange.API/Websockets/AbstractBinanceWebSocketClient.cs [hunk 1]
```diff
@@ -79,6 +79,22 @@ public Guid ConnectToDepthWebSocket(string symbol, BinanceWebSocketMessageHandle
...
             return CreateBinanceWebSocket(endpoint, messageEventHandler);
         }
+
+        /// <summary>
+        /// Connect to thePartial Book Depth Streams
+        /// </summary>
+        /// <param name="symbol"></param>
+        /// <param name="messageEventHandler"></param>
+        /// <returns></returns>
+        public Guid ConnectToPartialDepthWebSocket(string symbol,int levels, BinanceWebSocketMessageHandler<BinancePartialData> messageEventHandler)
```
Comment (BZ-CO) [inline BinanceExchange.API/Websockets/AbstractBinanceWebSocketClient.cs]: You can use `ConnectToDepthWebSocketCombinedPartial` method to do same, just use one symbol.

File: BinanceExchange.API/Websockets/AbstractBinanceWebSocketClient.cs [hunk 2]
```diff
@@ -79,6 +79,22 @@ public Guid ConnectToDepthWebSocket(string symbol, BinanceWebSocketMessageHandle
...
+        {
+            Guard.AgainstNullOrEmpty(symbol, nameof(symbol));
+            if (levels != 5 && levels != 10 && levels != 20)
```
Comment (glitch100) [inline BinanceExchange.API/Websockets/AbstractBinanceWebSocketClient.cs]: `const` out these levels into an array/list and check if it's in the list.

**Comment by glitch100** [comment]
Are these two issues delving into same area (levels)
https://github.com/glitch100/BinanceDotNet/pull/62/files

**Comment by N4rk0** [comment]
I added more commits to this branch, you should merge only up to 3f9a701

**Comment by N4rk0** [comment]
Replaced by [https://github.com/glitch100/BinanceDotNet/pull/66](url)

**Comment by N4rk0** [comment]
Replaced by [https://github.com/glitch100/BinanceDotNet/pull/66](url)

**Pull request closed without merging**